superdesk/data_updates/00007_20180321-092824_archive.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : tomas
+# Creation: 2018-03-21 09:28
+
+import os
+import subprocess
+import json
+from superdesk.commands.data_updates import DataUpdate
+
+def get_updated_editor_state(editor_state):
+    try:
+        p1 = subprocess.Popen(["echo", json.dumps(editor_state)], stdout=subprocess.PIPE)
+        p2 = subprocess.Popen(["node", os.path.abspath('../env/src/superdesk-core/superdesk/data_updates/00007_20180321-092824_archive.dist.js')], stdin=p1.stdout, stdout=subprocess.PIPE)
```

Comment:
normally you can use `__file__` to get module location, but I've made a quick try and it's not working with data updates (it depends how the script is launched). As an alternative, you can do following imports:
```python
import superdesk
from os.path import realpath, join, dirname
```
then do:

```python
join(dirname(realpath(superdesk.__file__)), "data_updates", "00007_20180321-092824_archive.dist.js")
```